src/dotnet/UI.Blazor.App/Components/ChatSearchBox/chat-search-box.css

```diff
@@ -63,10 +68,11 @@
     @apply bg-transparent;
     @apply text-03;
 }
-.chat-search-box label {
-    @apply relative;
-    @apply flex-x;
-}
+/*.chat-search-box label {*/
```

Comment:
why did you comment it? Shouldn't it me removed?

Reply:
oops, forgot to delete